spring-cloud-gcp-logging/src/main/java/org/springframework/cloud/gcp/logging/TraceIdLoggingWebMvcInterceptor.java

```diff
@@ -37,6 +40,11 @@
 
 	private static final String X_CLOUD_TRACE = "x-cloud-trace-context";
 
+	private static final String X_B3_TRACE = "X-B3-TraceId";
+
+	private static final List<String> TRACE_HEADERS = ImmutableList.of(X_CLOUD_TRACE,
```

Comment:
It's possible that both headers are set. What do we do in that case? Allow the precedence order to be configurable? Error out?

Also, I was thinking of an alternative design where for Zipkin we would have another implementation of `HandlerInterceptorAdapter`, and the user can choose which one (or both) to configure.